Arrowgene.Ddon.GameServer/Handler/GetCraftGradeupRecipeHandler.cs

```diff
@@ -0,0 +1,49 @@
+#nullable enable
+using Arrowgene.Ddon.GameServer.Characters;
+using Arrowgene.Ddon.Server;
+using Arrowgene.Ddon.Shared.Entity.PacketStructure;
+using Arrowgene.Ddon.Shared.Entity.Structure;
+using Arrowgene.Ddon.Shared.Model;
+using Arrowgene.Ddon.Shared.Network;
+using Arrowgene.Logging;
+using System;
+using System.Collections.Generic;
+
+namespace Arrowgene.Ddon.GameServer.Handler
+{
+    public class GetCraftGradeupRecipeHandler : GameStructurePacketHandler<S2CGetCraftGradeupRecipeRes>
```

Comment:
This should be `public class GetCraftGradeupRecipeHandler : GameStructurePacketHandler<C2SGetCraftGradeupRecipeReq>` 

s/S2CGetCraftGradeupRecipeRes/C2SGetCraftGradeupRecipeReq/

Reply:
Does this handler ever get called? I don't think you should be _receiving_ server responses in the server